sql/core/src/main/scala/org/apache/spark/sql/jdbc/OracleDialect.scala

```diff
@@ -29,7 +29,12 @@ private case object OracleDialect extends JdbcDialect {
   override def getCatalystType(
```

Comment:
Can you ask him to retry it in Spark 2.1? I am not sure whether he is using Apache Spark or the Spark released by other vendors.